pkgs/by-name/de/deepsource/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  installShellFiles,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "deepsource";
+  version = "0.8.6";
+
+  nativeBuildInputs = [ installShellFiles ];
+  src = fetchFromGitHub {
+    owner = "DeepSourceCorp";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-6uNb4cQVerrlW/eUkjmlO1i1YKYX3qaVdo0i5cczt+I=";
+  };
+
+  doCheck = false; # error when running unit test.
```

Comment:
Some or all of them? Can you see why (e.g. require certain services, require privileges, require network access)?

We can partially skip test with https://nixos.org/manual/nixpkgs/unstable/#ssec-skip-go-tests.

Reply:
hi @Aleksanaa updated